recipes-kernel/linux/linux-nilrt.inc

```diff
@@ -97,6 +97,8 @@ do_shared_workdir_append() {
 }
 
 pkg_postinst_ontarget_${KERNEL_PACKAGE_NAME}-base_append_x64 () {
+	[ ! -f /etc/natinst/safemode  ] || exit 0
```

Comment:
Couple nits:
1. could you switch this to positive logic, for readability? eg.
    ```bash
    [ -f /etc/natinst/safemode ] && exit 0
    ```
2. Could you add a comment to the script explaining that the postinst is insane to run in safmode, so that future readers don't wonder why the code is getting skipped?

Reply:
Done